package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "widget-tester",
-  "version": "1.11.4",
+  "version": "1.12.0",
```

Comment:
@ezequielc curious why bump to `12` instead of `11.5`? Thanks.

Reply:
@alex-deaconu Mainly to be extra-cautious given the error we saw in phantomJS with the regular update, before setting the fixed mocha version. Let me know if you think we should change to `11.5`. Thanks

Reply:
@ezequielc nope, let's leave as is. Thanks.